Public/Src/Tools/Xldb/XldbDataStore.cs

```diff
@@ -160,22 +159,23 @@ public IEnumerable<PipExecutionStepPerformanceReportedEvent> GetPipExecutionStep
         }
 
         /// <summary>
-        /// Gets process fingerprint computation events by key
+        /// Gets process fingerprint computation events by key. Commented out due to information being present
+        /// in Legacy Cache Miss Analyzer and not needed currently in DB

```

Comment:
I'm not sure that I'm getting 'the reasoning'

Reply:
No longer relevant as per Friday's meeting

---
In reply to: [321867862](https://github.com/microsoft/BuildXL/pull/836#discussion_r321867862) [](ancestors = 321867862)